test/run_test.sh

```diff
@@ -54,7 +58,7 @@ mv privcount.* old/ || true
 
 # Then run the injector, ts, sk, and dc
 echo "Launching injector, tally server, share keeper, and data collector..."
-privcount inject --simulate --port 20003 --log events.txt &
+xzcat events.txt.xz | privcount inject --simulate --port 20003 --log - &
```

Comment:
Some systems don't have xz installed by default, such as OS X.
Should we make this gzcat instead?